src/cpp/src/generation_config.cpp

```diff
@@ -96,7 +96,7 @@ void GenerationConfig::validate() const {
                     "Beam search with sampling is not supported yet. "
                     "Please either set do_sample=false to use beam search "
                     "or set num_beams=1 if you with to use multinomial sampling.");
-    OPENVINO_ASSERT(num_return_sequences <= num_beams, "num_return_sequences must be less or equal to num_beams");
+    OPENVINO_ASSERT(num_return_sequences > 0, "num_return_sequences must be greater than 0");
```

Comment:
Please, insert the lost parameters from here: https://github.com/ilya-lavrenov/openvino.genai/blob/e7562739ecfd1407d67cedd28d8933f791498d51/text_generation/causal_lm/cpp/continuous_batching/library/src/generation_config.cpp#L22